app/models/user.rb

```diff
@@ -33,6 +33,7 @@ def field_order
                         length: { maximum: 24 }
   validates :gender, inclusion: { in: %w(female male) }, allow_nil: true
   validates :age_range, inclusion: { in: VALID_AGE_RANGES }, allow_nil: true
+  validates :contact_number, format: { with: /\A0[12379]\d{9}\z/, message: 'is an invalid UK number', if: 'contact_number.present?'}, allow_blank: true
```

Comment:
Hi @asmega the translation has been added for English and Welsh. I will squash it if you are happy with it. 

https://github.com/moneyadviceservice/frontend/commit/70eb9455bb625c348d0b4e088cb0982eab6c47b1